easybuild/easyconfigs/s/SimpleElastix/SimpleElastix-1.1.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -74,6 +74,9 @@ components = [
         'source_urls': ['https://github.com/InsightSoftwareConsortium/ITK/archive/'],
         'checksums': ['c6b3c33ecc73104c906e0e1a1bfaa41a09af24bf53a4ec5e5c265d7e82bdf69f'],
         'srcdir': '%%(builddir)s/ITK-%s' % local_itk_ver,
+        # we need to take matters into our own hands here to ensure we have a unique build directory per component;
```

Comment:
I'm wondering if it wouldn't be better if this is done in the CMakeMake EB. Currently the build folders are `%(start_dir)s/easybuild_obj` for these components, aren't they? How about instead if using `%(builddir)s/easybuild_obj` we use `%(builddir)s/easybuild_obj_%(name)s` or so?

Reply:
Hmm, that's also an option I guess, yeah...

But let's make that a separate PR, I don't want to block https://github.com/easybuilders/easybuild-easyblocks/pull/1933 over this.